### PR TITLE
docs(ecosystem): adds fastify-hana to the community plugins list

### PR DESCRIPTION
**Pull request by yoav0gal**
Adds [fastify-hana](https://github.com/yoav0gal/fastify-hana) to the community plugins list

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: docs/Guides/Ecosystem.md [hunk 1]
```diff
@@ -386,6 +386,8 @@ section.
   Providers.
 - [`fastify-guard`](https://github.com/hsynlms/fastify-guard) A Fastify plugin
   that protects endpoints by checking authenticated user roles and/or scopes.
+- [`fastify-hana`](hhttps://github.com/yoav0gal/fastify-hana) connects your
```
Comment (Uzlopak) [inline docs/Guides/Ecosystem.md:389]:
```suggestion
- [`fastify-hana`](https://github.com/yoav0gal/fastify-hana) connects your
```

**Review by mcollina — APPROVED** [review]
lgtm

File: docs/Guides/Ecosystem.md (hunk 1 above)
Comment (yoav0gal) [reply docs/Guides/Ecosystem.md]: ty

**Review by Fdawgs — APPROVED** [review]

**Review by Uzlopak — APPROVED** [review]
LGTM

**Pull request merged**